packages/dx-react-grid/docs/guides/editing.md

```diff
@@ -48,12 +48,14 @@ Note, you can also use the `onAddedRowsChange` event to initialize a created row
 
 End-users can click cells and thus activate in-place editors, provided that you include the `TableInlineCellEditing` plugin into the Grid's configuration. To save changes, they can move focus or press Enter. Esc discards changes and deactivates the cell editor.
```

Comment:
```suggestion
Users can click cells and activate in-place editors if you include the `TableInlineCellEditing` plugin in the Grid's configuration. To save changes, they can move focus or press Enter. Esc discards changes and deactivates the cell editor.
```

Reply:
Looks like clicking cells and activating in-place editors are two different actions, while in fact users activate the editors by clicking the cells.

Reply:
```suggestion
End-users can click cells to activate in-place editors if you include the `TableInlineCellEditing` plugin in the Grid's configuration. They can move focus or press Enter to save changes, or press Esc to discard the changes and deactivate the cell editor.